pallas-traverse/src/lib.rs

```diff
@@ -97,7 +97,10 @@ pub enum Error {
     UnknownCbor(String),
 
     #[error("Unknown era tag: {0}")]
-    UnkownEra(u16),
+    UnknownEra(u16),
+
+    #[error("Invalid era for request: {0}")]
+    InvalidEra(u16),
```

Comment:
I think we could use the `crate::Era` struct for this particular error since it will be thrown from methods where the Era of the data is strictly known (the `UnknownEra` relies on u16 because it is used for errors during the parsing of the value).

Reply:
fixed

Reply:
fixed